api/src/pcapi/domain/user_emails.py

```diff
@@ -120,7 +117,7 @@ def send_booking_cancellation_emails_to_user_and_offerer(
     reason: BookingCancellationReasons,
 ) -> None:
     if reason == BookingCancellationReasons.BENEFICIARY and booking.individualBooking is not None:
-        send_individual_booking_cancellation_email(booking.individualBooking)
+        send_beneficiary_booking_cancellation_email(booking.individualBooking)
```

Comment:
est-ce qu'on peut être un peu plus clair dans le wording ? 
car là on peut penser que c'est une annulation par un bénéficiaire
alors que la distinction importante c'est : est-ce qu'on l'envoie au bénéficiaire ou à l'offreur.
`send_booking_cancellation_to_beneficiary_email` ?
idem pour nom du template, tag, fichier, etc

Reply:
enfin il y a aussi une distinction entre le destinataire : offerer ou beneficiary ?
pas de distinction c'est du : beneficiary to beneficiary.